### PR TITLE
chore(zh-cn): run autocorrect to fix text layout Part 4

### PR DESCRIPTION
**Pull request by yin1999**
### Description

run autocorrect to fix text layout Part 4

### CODE REVIEW

File: files/zh-cn/web/http/headers/index.md [hunk 1]
```diff
@@ -379,7 +379,7 @@ _[在此处](/zh-CN/docs/Glossary/CORS)了解更多关于 CORS。_
 - {{HTTPHeader("Signed-Headers")}} {{experimental_inline}}
   - : [`Signed-Headers`](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#rfc.section.5.1.2) 标头字段标识要包含在签名中的响应标头字段的有序列表。
 - {{HTTPHeader("Server-Timing")}}
-  - : 传达给定请求-响应周期的一个或多个指标和描述。
+  - : 传达给定请求 - 响应周期的一个或多个指标和描述。
```
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/index.md:382]: why this is different from line 59 🤔 

```suggestion
  - : 传达给定请求—响应周期的一个或多个指标和描述。
```

Comment (yin1999) [reply files/zh-cn/web/http/headers/index.md:382]: Didn't notice the change on this line, the modification on line 59 was manually modified by myself (`autocorrect` just gave similar changes to this line).

File: files/zh-cn/web/javascript/reference/global_objects/string/index.md [hunk 2]
```diff
@@ -202,7 +202,7 @@ otherwise my code is unreadable.";
 
 然而，整个 Unicode 字符集比 65536 大得多。额外的字符作为*代理对*（surrogate pairs）存储在 UTF-16 中，代理对是表示单个字符的 16 位码元对。为了避免起义，该对的两个部分必须介于 `0xD800` 和 `0xDFFF` 之间，并且这些码元不用于对单码元（single-code-unit）进行编码。因此，“单独的代理项”通常不是操作字符串的有效值——例如 [`encodeURI()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/encodeURI) 将为单独的代理项抛出 {{jsxref("URIError")}}。每个 Unicode 字符由一个或者两个 UTF-16 码元组成，也称为 _Unicode 码位_（codepoint）。每个 Unicode 码位都可以使用 `\u{xxxxxx}` 写成一个字符串，其中 `xxxxxx` 表示 1–6 个十六进制数字。
 
-在 Unicode 字符之上，有一些 Unicode 字符序列应该被视为一个视觉单元，被称为*字素簇*（grapheme cluster）。更通常的案列是 emoji：很多具有一系列变化的 emoji 实际上是由多个 emoji组成的，通常由 \<ZWJ>（`U+200D`）字符连接。
+在 Unicode 字符之上，有一些 Unicode 字符序列应该被视为一个视觉单元，被称为*字素簇*（grapheme cluster）。更通常的案列是 emoji：很多具有一系列变化的 emoji 实际上是由多个 emoji 组成的，通常由 \<ZWJ>（`U+200D`）字符连接。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/string/index.md:205]:
```suggestion
在 Unicode 字符之上，有一些 Unicode 字符序列应该被视为一个视觉单元，被称为*字素簇*（grapheme cluster）。更通常的案例是 emoji：很多具有一系列变化的 emoji 实际上是由多个 emoji 组成的，通常由 \<ZWJ>（`U+200D`）字符连接。
```

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**